### DISCUSSION TITLE
Missing Menu Bar Buttons- ability to control what is on the menu bar?

### DISCUSSION THREAD

**Discussion by iamnacho** [Aris-t2/CustomCSSforTb#18] [General]
Hello,
I'm working on upgrading to TB115 from TB102. After the upgrade (with or without CustomCSSforTB) I have no menu buttons.

![tb_no_buttons](https://github.com/Aris-t2/CustomCSSforTb/assets/11319791/73f151d4-7a24-4213-8972-bb3a3669b8d9)

Is there a way to select which buttons are placed on this menu bar either using CustomCSSforTB or with userChrome.css?  I can put the buttons there manually by right-clicking and Customizing , but I help manage about 100 Thunderbird installs and it would be nice to do it "programmatically" vs. Manually.

I used the inspector tool and think the buttons might be named like "get-message" "write-messages", etc. but I'm really at a loss on how to create the CSS to have them show.

Any help is appreciated!

**Comment by Kravimir** [comment]
To fix missing toolbar buttons, try deleting the "xulstore.json" file from your profile. Once you have the buttons set how you want, try copying the new version of that same file to the profiles on the other workstations. (Renaming the old version instead of overwriting it is a good precaution.)

More info:
https://support.mozilla.org/en-US/kb/profiles-where-thunderbird-stores-user-data
http://kb.mozillazine.org/Files_and_folders_in_the_profile_-_Thunderbird

**Comment by iamnacho** [comment]
Thank you Kravimir,  I'll give that a shot!
